openapi/openapi_v2_resource_test.go

```diff
@@ -181,6 +181,931 @@ func TestNewSpecV2Resource(t *testing.T) {
 	})
 }
 
+func TestBuildResourceName(t *testing.T) {
+
+	testCases := []struct {
+		path                 string
+		expectedResourceName string
+		expectedError        error
+	}{
+		{
+			path:                 "/cdns",
+			expectedResourceName: "cdns",
+			expectedError:        nil,
+		},
+		{
+			path:                 "/v1/cdns",
+			expectedResourceName: "cdns_v1",
+			expectedError:        nil,
+		},
+		{
+			path:                 "/v11/cdns",
+			expectedResourceName: "cdns_v11",
+			expectedError:        nil,
+		},
+		{
+			path:                 "/v1.1.1/cdns",
+			expectedResourceName: "cdns", //TODO: possible bug
```

Comment:
bug?

Reply:
semver in paths is not supported at the moment, put a comment instead to make it clear. this is not a bug, it's expected to work like that